src/main/resources/db/scripts/V7_CREATE_ORDER_TABLE.SQL

```diff
@@ -0,0 +1,8 @@
+CREATE TABLE order (
+    orderId INT PRIMARY KEY AUTO_INCREMENT,
+    userId INT NOT NULL,
+    orderDate TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    deliveryDate TIMESTAMP DEFAULT CURRENT_TIMESTAMP,
+    totalAmount DECIMAL(10, 2) NOT NULL,
+    addressId TEXT NOT NULL,
```

Comment:
Column name is addressId. Instead it should be address. Fix it and mark this conversation as resolved.